metricbeat/module/kubernetes/controllermanager/_meta/fields.yml

```diff
@@ -0,0 +1,124 @@
+- name: controllermanager
+  type: group
+  description: >
+    Controller manager metrics
+  release: ga
+  fields:
+    - name: handler
+      type: keyword
+      description: >
+        Request handler
+    - name: code
+      type: keyword
+      description: >
+        HTTP code
+    - name: method
+      type: keyword
+      description: >
+        HTTP method
+    - name: host
+      type: keyword
+      description: >
+        Request host
+    - name: name
+      type: keyword
+      description: >
+        Name for the resource
+    - name: zone
+      type: keyword
+      description: >
+        Infrastructure zone
+    - name: process
+      type: group
+      fields:
+        - name: cpu.sec
+          type: double
+          description: CPU seconds
+        - name: memory.resident.bytes
+          type: long
+          description: Bytes in resident memory
+        - name: memory.virtual.bytes
+          type: long
+          description: Bytes in virtual memory
+        - name: fds.open.count
+          type: long
+          description: Number of open file descriptors
+        - name: started.sec
+          type: double
+          description: Seconds since the process started
+    - name: http
+      type: group
+      fields:
+        - name: request.duration.us.percentile.*
+          type: object
+          object_type: double
+          description: Request duration microseconds percentiles
+        - name: request.duration.us.sum
+          type: double
+          description: Request duration microseconds cumulative sum
+        - name: request.duration.us.count
+          type: long
+          description: Request count for duration
+        - name: request.size.bytes.percentile.*
+          type: object
+          object_type: long
+          description: Request size percentiles
+        - name: request.size.bytes.sum
+          type: long
+          description: Request size cumulative sum
+        - name: request.size.bytes.count
+          type: long
+          description: Request count for size
+        - name: response.size.bytes.percentile.*
+          type: object
+          object_type: long
+          description: Response size percentiles
+        - name: response.size.bytes.sum
+          type: long
+          description: Response size cumulative sum
+        - name: response.size.bytes.count
+          type: long
+          description: Response count
```

Comment:
You can add `format: bytes` to all these fields containing bytes.

Reply:
@jsoriano  can you confirm `bytes` is the right type?
at elasticsearch reference bytes is intended for 8 bit numbers

Reply:
`format: bytes` declares the [formatter for Kibana](https://www.elastic.co/guide/en/kibana/7.1/field-formatters-numeric.html), this is apart of the type, that is probably fine as `long`. You can find some examples of the use of `format` in other modules.